app/workers/import_worker.rb

```diff
@@ -71,4 +73,18 @@ def process_follows
       end
     end
   end
+
+  def process_favourites
+    import_rows.each do |row|
+      begin
+        domain, account, id = row
+
+        status = (domain == Rails.configuration.x.local_domain) ? Status.find(id) : FetchRemoteStatusService.new.call("http://#{domain}/users/#{account}/#{id}.atom")
```

Comment:
Yeah this doesn't work with any status that doesn't come from a Mastodon instance. Exporting URLs would make this code simpler.